packages/angular_devkit/core/src/virtual-fs/path.ts

```diff
@@ -5,7 +5,7 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-import { BaseException } from '@angular-devkit/core';
+import { BaseException } from '..';
```

Comment:
I like the idea of the rule but forcing a change such as this, I think this causes a decrease in readability and overall code quality.  The rule should at a minimum allow the name of the package itself and should ideally support any path mapping.  The second part especially as the rule pretty much makes advanced path mapping unusable.